deploy/frontend.yaml

```diff
@@ -44,6 +44,14 @@ objects:
               title: "Advanced Cluster Security"
               href: "/application-services/subscriptions/rhacs"
               product: "Subscription Watch"
+            - appId: "subscriptions"
+              title: "RHEL Subscriptions"
```

Comment:
apologies, almost missed this... 2 questions @san7ket 

- There appears to be a `RHEL` option that is duplicated around line 27-30, is this intentional?
- And we just removed `RHOSAK` or `streams` do we also want to remove it here?

Reply:
We want to keep "RHEL" and "RHEL Subscriptions": , ephemeral UI is not intelligent so we need to mentioned both, as our tests needs both to be present.

RHOSAK we will drop if we have already dropped from stage

Reply:
Yes, RHOSAK was dropped from prod and stage as of the GUI release last week, [v3.2.0](https://github.com/RedHatInsights/curiosity-frontend/blob/main/CHANGELOG.md#320-2023-08-15), so if you remove it that should be the last change